pom.xml

```diff
@@ -1517,7 +1517,7 @@
     <log4j.version>1.2.17</log4j.version>
     <mockito-core.version>2.28.2</mockito-core.version>
     <!--Internally we use a different version of protobuf. See hbase-protocol-shaded-->
-    <external.protobuf.version>2.5.0</external.protobuf.version>
+    <external.protobuf.version>3.11.4</external.protobuf.version>
```

Comment:
Ouch.

Do you get the bit where we use shaded pb3 internally but that unshaded pb2.5 is available for users? (HBase3/master branch is looking like it will purge the unshaded pb2.5 usage... but that is another PR).

Reply:
The idea is to upgrade PB2.5 to PB3.x in HBase 3.x.
Now is the best moment to do this upgrade.

Reply:
HBASE-16761 was one suggestion. Another, more recent one is no PB2.5 or hbase-protocol module in hbase3.  See HBASE-23797.

Reply:
Just let me know your preference and I will try to adapt this PR, or even close it and open a new one.